wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/ConversationMessageActionController.swift

```diff
@@ -101,16 +101,18 @@ final class ConversationMessageActionController {
         case .forward:
             return message.canBeForwarded
         case .like:
-            return message.canBeLiked && !message.liked
+            return message.canAddReaction && !message.liked
         case .unlike:
-            return message.canBeLiked && message.liked
+            return message.canAddReaction && message.liked
         case .resend:
             return message.canBeResent
         case .showInConversation:
             return context == .collection
         case .sketchDraw,
              .sketchEmoji:
             return message.isImage
+        case .react:
```

Comment:
If you have introduced a new message action "react", why do we need to keep "like" and "unlike"?

Reply:
removed